cpp/proto/serializer.cc

```diff
@@ -1,8 +1,16 @@
 /* -*- indent-tabs-mode: nil -*- */
 #include "proto/serializer.h"
 
+// TODO(alcutter): Separate out the various serialisation modules so that
+// servers only need to link the one(s) they want to use.
+extern "C" {
```

Comment:
Actually no, I've got a different PR on Ben's project to fix other stuff; I'll open another one!